docbook/src/docbkx/autoscale-devguide.xml

```diff
@@ -70,6 +70,16 @@
         </legalnotice>
         <revhistory>
             <revision xmlns:raxm="http://docs.rackspace.com/api/metadata">
+                <date>2015-03-17</date>
+                <revdescription>
+                    <itemizedlist spacing="compact">
+                        <listitem>
+                            <para>Updated <link linkend="identityEndpoints">Identity endpoints </link>section. Changed launchConfiguration loadBalancerId attribute type to "String."</para>
```

Comment:
The link works; the markup just includes the space inside the link, which happens nowhere else.

Nope, not run-on sentences; actual width of lines in the XML. Sometimes, paragraphs are wrapped; sometimes they are not. I was wondering if that's intentional, and if there's some guideline behind it.